### DISCUSSION TITLE
in VS Code How can I make settings for the Extension ?

### DISCUSSION THREAD

**Discussion by paladio** [yandeu/five-server#56] [Help / Questions]
Thanks for this valuable Extension , in VS Code How can I make settings for the Extension ? 
without installing npm version and just with install on vs code ?

**Comment by yandeu** [comment]
In VSCode you can adjust some settings.

But you can also add a `fiveserver.config.js` file per project with the setting from [/src/types.ts](https://github.com/yandeu/five-server/blob/main/src/types.ts)

**Reply by paladio** [reply]
Is it possible to put the settings in the **settings.json** in the folder **.vscode** and add it there?
Which settings can be changed from within the VS Code and where can this change be made?
